test/test_util_pipe.py

```diff
@@ -71,6 +72,7 @@ def test_pipe_protocol(self):
 
         __pool = concurrent.futures.ThreadPoolExecutor()
         for count in range(100):
+            log.debug("Iteration no. %d", count + 1)
```

Comment:
`log.*` methods aren't meant to be used in tests. This file is the only one using them.